e2e-tests/functions

```diff
@@ -80,13 +80,25 @@ version_gt() {
 	fi
 }
 
+get_proxy_size() {
+	local cluster=${1}
+	if [[ "$(kubectl_bin get pxc "${cluster}" -o 'jsonpath={.spec.haproxy.enabled}')" == "true" ]]; then
+		kubectl_bin get pxc "${cluster}" -o 'jsonpath={.spec.haproxy.size}'
```

Comment:
It will not work because e.g. in some tests, we run only one proxysql pod but the operator scales up it to two automatically.